src/tests/ftest/soak/soak.yaml

```diff
@@ -5,24 +5,32 @@ hosts:
         - boro-C
         - boro-D
     client_partition: daos_client
-
-timeout: 7200
+# This timeout must be longer than the test_timeout param (+30minutes)
+timeout: 5400
 logdir: /tmp/soak
 server_config:
     name: daos_server
+    log_mask: ERR
+    crt_timeout: 2000
```

Comment:
Any specific reason why so long CRT_TIMEOUT is needed?

Reply:
I had to bump this to 2000 when running PSM2.